keras_cv/models/efficientnet_lite.py

```diff
@@ -507,14 +553,16 @@ def EfficientNetLiteB0(
     classifier_activation="softmax",
     **kwargs,
 ):
+    """Instantiates the EfficientNetLiteB0 architecture."""
```

Comment:
These docstrings are added at the end of the file, and include the full EfficientNet docstring, so let's remove these new mini ones.

Reply:
Done.